### PR TITLE
Display folder subtitle when inside the organization file

### PR DESCRIPTION
**Pull request by LunarX**

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/ui/fileList/FileListViewModel.kt [hunk 1]
```diff
@@ -267,6 +267,13 @@ class FileListViewModel(application: Application) : AndroidViewModel(application
         cancelDownloadFiles()
     }
 
+    fun shouldDisplaySubtitle(folderId: Int, userDrive: UserDrive?): Boolean {
+        FileController.getFileById(folderId, userDrive)?.let {
+            return it.getVisibilityType() == File.VisibilityType.IS_TEAM_SPACE
+        }
+        return false
+    }
```
Comment (tevincent) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/FileListViewModel.kt:275]: Maybe we can improve like this?

```kotlin
fun shouldDisplaySubtitle(folderId: Int, userDrive: UserDrive?): Boolean {
        return FileController.getFileById(folderId, userDrive)?.let {
            it.getVisibilityType() == File.VisibilityType.IS_TEAM_SPACE
        } ?: false
    }
```

File: app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt [hunk 2]
```diff
@@ -275,6 +276,23 @@ open class FileListFragment : MultiSelectFragment(MATOMO_CATEGORY), SwipeRefresh
         multiSelectLayout?.toolbarMultiSelect?.removeInsets()
     }
 
+    private fun setPublicFolderSubtitle() {
+        val driveName = AccountUtils.getCurrentDrive()?.driveAccount?.name
```
[the remark this answers is not in the document]
Comment (LunarX) [reply app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt:280]: I thought I had it declared right after `setToolbarTitle` but I might have mistaken it for `setupToolbars` when placing my new function. I moved it back right after the correct `setToolbarTitle()` this time

File: app/src/main/java/com/infomaniak/drive/ui/fileList/FileListViewModel.kt (hunk 1 above)
Comment (LunarX) [reply app/src/main/java/com/infomaniak/drive/ui/fileList/FileListViewModel.kt:275]: An even easier to read one could be:
```kt
val folder = FileController.getFileById(folderId, userDrive) ?: return false
return folder.getVisibilityType() == File.VisibilityType.IS_TEAM_SPACE
```

File: app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt [hunk 3]
```diff
@@ -275,6 +276,23 @@ open class FileListFragment : MultiSelectFragment(MATOMO_CATEGORY), SwipeRefresh
...
+        val folderId = folderId
+        val userDrive = userDrive
```
[the remark this answers is not in the document]
Comment (LunarX) [reply app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt:282]: Yeah no they're not needed

**Review by tevincent — APPROVED** [review]

**Pull request merged**